src/portfft/dispatcher/workgroup_dispatcher.hpp

```diff
@@ -108,6 +108,14 @@ PORTFFT_INLINE void workgroup_impl(const T* input, T* output, const T* input_ima
   const IdxGlobal input_distance = kh.get_specialization_constant<detail::SpecConstInputDistance>();
   const IdxGlobal output_distance = kh.get_specialization_constant<detail::SpecConstOutputDistance>();
 
+#ifdef PORTFFT_USE_SCLA
+  T wi_private_scratch[detail::SpecConstWIScratchSize];
+  T priv[detail::SpecConstNumRealsPerFFT];
+#else
+  T wi_private_scratch[2 * wi_temps(detail::MaxComplexPerWI)];
+  T priv[2 * MaxComplexPerWI];
+#endif
```

Comment:
`priv` can be renamed something more descriptive.

Reply:
We have used priv throughout the codebase,
changing it here would also warrant changes in other implementations, workitem and subgroup, so I say let's leave it as priv for now ?

Reply:
I see your point about `priv` being used in lots of places, but we can still come up with a better name for it here, and change it just within this limited scope. `priv` is undescriptive and its now even the only work-item private memory we're allocating!

Reply:
I'm not sure we can find a much better name since this is generic and used in a couple of different places from what I remember.

Reply:
I consider myself outvoted!